pkg/frontend/adminactions/kubeactions.go

```diff
@@ -30,6 +31,9 @@ type KubeActions interface {
 	KubeDelete(ctx context.Context, groupKind, namespace, name string) error
 	CordonNode(ctx context.Context, nodeName string, unschedulable bool) error
 	DrainNode(ctx context.Context, nodeName string) error
+	RunCertificateApprove(ctx context.Context, csrName string) error
+	RunCertificateMassApprove(ctx context.Context) error
+	RunCertificateApprovalUpdate(ctx context.Context, csr *certificatesv1.CertificateSigningRequest) error
```

Comment:
naming things is hard.  I think these can be named differently, but obviously it's very subjective so whatever you thing :smile: 
```suggestion
	ApproveCsr(ctx context.Context, csrName string) error
	ApproveAllCsrs(ctx context.Context) error
	updateCsr(ctx context.Context, csr *certificatesv1.CertificateSigningRequest) error